node/t3rn-parachain/src/chain_spec.rs

```diff
@@ -117,25 +118,25 @@ pub fn polkadot_config() -> ChainSpec {
                 // Invulnerable collators
                 vec![
                     (
-                        get_account_id_from_adrs(
-                            "5FKjxoi5Yfjwa1aXesFWRXMpvs4vJMXFeG2ydFPyNwUn4qiW",
-                        ),
-                        get_aura_id_from_adrs("5FKjxoi5Yfjwa1aXesFWRXMpvs4vJMXFeG2ydFPyNwUn4qiW"),
+                        hex!("5232d5d6b3904523020c08addf5b648f5ecb1e3481c04fe46d2d82efb193b674")
```

Comment:
suggest we switch back to identifying invulnerables and endowed accounts using their SS58 addresses - a little handier for double checking and when distributing the chain spec